families/supplychain/postgreSQL/tables_sample_data.sql

```diff
@@ -0,0 +1,100 @@
+-- Definition of tables for Supply Chain DB
+-- Tables to create
+
+
+CREATE TABLE block (
+	block_id			char(128) primary key,		-- primary key
```

Comment:
This was just messiness on my part, from when I was developing the database. Removed.